src/f4enix/input/elite.py

```
@@ -138,261 +139,215 @@ def extract_sector(
+                        # if so, apply it and check if it's a plane parallel to
+                        # the boundary
+                        tr_name = "TR" + str(tr)
+                        trans = inp.transformations[tr_name]
+                        # check if it's only a translation
+                        # if len(trans.values) < 12:
+                        #     continue
+                        # if it's a rotation, rotate the coefficients
+                        p_coeffs = _rotate_plane(trans, p_coeffs)
+                    # if not, just check if it's parallel to the boundary
+                    # if the plane is similar to boundary planes, modify it
+                    if _check_tol(tol, list(zip(p_coeffs, coeffs))):
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)